freeipa/src/test/java/com/sequenceiq/freeipa/service/freeipa/user/UsersyncPollerTest.java

```diff
@@ -23,7 +23,6 @@
 import com.sequenceiq.freeipa.entity.Stack;
 import com.sequenceiq.freeipa.entity.UserSyncStatus;
```

Comment:
Can we have tests that make sure that force works for both actor sync and full syncs?